source/braille.py

```diff
@@ -304,6 +304,11 @@
 )
 SELECTION_SHAPE = 0xC0 #: Dots 7 and 8
 
+#: The braille shape shown on a braille display when
+#: the number of cells used by the braille handler is lower than the actual number of cells.
+#: The 0 based position of the shape is equal to the number of cells used by the braille handler.
+END_OF_BRAILLE_OUTPUT_SHAPE = 0xFF  # All dots

```

Comment:
For docstrings for variables, I suggest using this syntax, which is supported by Sphinx and epydoc.
The syntax used here does not give hinting in VS Code, but the epydoc syntax works in VS Code and PyCharm.

https://stackoverflow.com/questions/8820276/docstring-for-variable

```suggestion
END_OF_BRAILLE_OUTPUT_SHAPE = 0xFF  # All dots
"""
The braille shape shown on a braille display when
the number of cells used by the braille handler is lower than the actual number of cells.
The 0 based position of the shape is equal to the number of cells used by the braille handler.
"""
```